GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -41,34 +59,41 @@ private enum AssetCatalog: String {
 extension AssetsCatalogParser {
   static let imageSetExtension = "imageset"
 
-  fileprivate func processCatalog(items: [[String: AnyObject]], withPrefix prefix: String = "") {
+  fileprivate func process(items: [[String: Any]], withPrefix prefix: String = "") -> [Entry] {
+    var result = [Entry]()
+
     for item in items {
       guard let filename = item[AssetCatalog.filename.rawValue] as? String else { continue }
```

Comment:
I totally forgot to see that in the other PR about actool migration, but if you're only using the `enum AssetCatalog` as a way to group constants together (and never use the `cases` other than via their `rawValue` to get the String constant), I'd suggest you to use an `enum` with no cases, but with `static let` constants namespaced in the enum instead:

```swift
private enum AssetCatalogKeys {
  static let children = "children"
  static let filename = "filename"
  static let providesNamespace = "provides-namespace"
  static let root = "com.apple.actool.catalog-contents"
}
```